zebra-consensus/src/block.rs

```diff
@@ -206,12 +206,18 @@ where
             // Finally, submit the block for contextual verification.
             let new_outputs = Arc::try_unwrap(known_utxos)
                 .expect("all verification tasks using known_utxos are complete");
+            let block_value_balance = block
+                .chain_value_pool_change(&transparent::utxos_from_ordered_utxos(
+                    new_outputs.clone(),
+                ))
+                .expect("all utxos needed to compute value balance should be known at this stage");
```

Comment:
Which UTXOs are in `new_outputs`, and which ones does `chain_value_pool_change` require?

(Check the documentation for both functions.)

Reply:
Each call we have to `chain_value_pool_change` will fail at some test or another.

So i replaced all calls we have (3) with `ValueBalance::zero()` and a `TODO` comment on each. This should allow us to potentially merge this PR if we want.

https://github.com/ZcashFoundation/zebra/pull/2599/commits/911921fa2ec7bcbb56efa94849ad9e4026a675fb

Reply:
If we merge this PR with incorrect balances, we'll need to do another state version update when we fix the balances.

So let's try to put the correct balances in the state in this PR.
I'll make some other suggestions about how to do that.